f8a_worker/workers/githubstats.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import logging
+from f8a_worker.base import BaseTask
+
+logger = logging.getLogger(__name__)
+
+class GithubStats(BaseTask):
+    """ Collects statistics by parsing Github Page """
+    _analysis_name = "github_manifest_details"
+
+    @classmethod
+    def _get_page(cls, repo_name):
+        github_url = 'https://github.com/'
+        url = github_url + "/" + repo_name
+        raw_data = requests.get(url)
+        if raw_data:
+            soup = BeautifulSoup(raw_data.text, "html.parser")
+            return soup
+        else:
+            logger.info("Could not fetch the page for repo_name:"% repo_name)
+
+    @classmethod
+    def _get_stargazers(cls, soup, repo_name):
+        stars =  soup.find(attrs={"href": repo_name + "/" + "stargazers"})
+        return (stars.text).strip() if stars is not None else 0
+
+    @classmethod
+    def _get_watchers(cls, soup, repo_name):
+        watches =soup.find(attrs={"href": repo_name + "/" + "watchers"})
```

Comment:
Can be simplified a bit: `{"href": repo_name + "/watchers"}`

Reply:
updated w.r.to suggestions.